airflow/executors/base_executor.py

```diff
@@ -54,9 +54,9 @@ def queue_command(self, task_instance, command, priority=1, queue=None):
         key = task_instance.key
         if key not in self.queued_tasks and key not in self.running:
             self.log.info("Adding to queue: %s", command)
-            self.queued_tasks[key] = (command, priority, queue, task_instance)
         else:
-            self.log.info("could not queue task {}".format(key))
+            self.log.info("Adding to queue even though already queued or running {}".format(command, key))
+        self.queued_tasks[key] = (command, priority, queue, task_instance)
```

Comment:
Does this make any effect? if queued_task has the key, you are just effectively refreshing the value, not sure about the motivation for this change.

Reply:
The reason I did this is in case the task was in the running state. It doesn't hurt even for the queued state becase task instance/etc might have changed, and I wanted to minimize the size of the fork (i.e. could have broken out the if state == running then queue the task into a separate elif).